docs/docs/api/MockPool.md

```diff
@@ -81,7 +81,7 @@ By default, `reply` and `replyWithError` define the behaviour for the first matc
 
 ### Return: `MockScope`
 
-A `MockScope` is associated with a single `MockInterceptor`. With this, we can configure the default behaviour of a intercepted reply.
+A `MockScope` is associated with a single `MockInterceptor`. With this, we can configure the default behaviour of an intercepted reply.
```

Comment:
@NathanBaulch I also do random PR-s with `typos`.
How do you spot wrong indefinite articles?

Reply:
IntelliJ highlights them. I usually only fix them if the PR I'm working on isn't getting too out of hand.

Reply:
https://www.jetbrains.com/help/idea/grammar.html